main.scss

```diff
@@ -1,14 +1,173 @@
+// sass-lint:disable no-qualifying-elements nesting-depth no-important class-name-format
+
```

Comment:
Lets add a comment at the top if this file explaining how it gets used as its not a normal use case. Explaining about how it only gets used via the build service and the url for this is in the Coral admin

Reply:
Thanks for the reminder. This is done now.